src/codegen/codegen.ts

```diff
@@ -252,6 +255,21 @@ export function generateRequestParams(request: ProxyData['request']): string {
   `
 }
 
+export function generateParameterizationCustomCode(rules: TestRule[]): string {
```

Comment:
Open to feedback about this approach.

Reply:
What do you think about filtering out disabled rules in `generateScript`?
We can also change the function signature to only accept parameterization rules to make it simpler (though the filtering code will just move elsewhere 🤷‍♂️ )
```ts
function generateParameterizationCustomCode(rules: ParameterizationRule[]): string {
  // ...
}
```